ga4gh/backend.py

```diff
@@ -803,6 +874,12 @@ def __init__(
                 numAlignments=numAlignments)
             self.addDataset(dataset)
 
+        # g2pDatasets
+        self._g2pDataset = genotype_phenotype.\
+            G2PDataset('cgd',
```

Comment:
This isn't empty...  We should really have a list of G2P datasets (or G2pAnnotationSets, or whatever we want to call them). The semantics of this sort of stuff is a lot easier then.

We can't have paths referring to the git structure in the production code, as these will all break when the package is installed.